sections/testingandquality/avoid-global-test-fixture.md

```diff
@@ -15,7 +15,9 @@ it('When updating site name, get successful confirmation', async () => {
   const siteUnderTest = await SiteService.addSite({
     name: 'siteForUpdateTest'
   });
+

```

Comment:
@rluvaton Why not AAA it appropriately? Shall we create AAA eslint rule?

AAA example:

![image](https://user-images.githubusercontent.com/8571500/109857287-32c04500-7c63-11eb-927e-f54fc0a79d34.png)

Reply:
Is my last commit resolve what you requested?

Reply:
Yes! 🔥